graphics/graphite2/files/patch-src-CMakeLists.txt.diff

```diff
@@ -0,0 +1,14 @@
+Don't use -mfpmath=sse -msse2 on Mac OS universal ARM/Intel, PowerPC/Intel
+builds.
+https://github.com/silnrsi/graphite/pull/84
+--- src/CMakeLists.txt.orig	2020-04-01 04:53:13.000000000 +0200
++++ src/CMakeLists.txt	2023-03-23 14:12:51.000000000 +0100
+@@ -107,7 +107,7 @@
+         COMPILE_FLAGS   "-Wall -Wextra -Wno-unknown-pragmas -Wendif-labels -Wshadow -Wctor-dtor-privacy -Wnon-virtual-dtor -fno-rtti -fno-exceptions -fvisibility=hidden -fvisibility-inlines-hidden"
+         LINK_FLAGS      "-nodefaultlibs ${GRAPHITE_LINK_FLAGS}"
+         LINKER_LANGUAGE C)
+-    if (${CMAKE_SYSTEM_PROCESSOR} MATCHES "x86|i.86")
```

Comment:
Isn’t this a wrong place though? `if  (${CMAKE_SYSTEM_NAME} STREQUAL "Darwin")` is lower down, line 110 is under `if  (${CMAKE_SYSTEM_NAME} STREQUAL "Linux")`.

Reply:
oh sh**. The upstream PR looks ok, though. Will update soon.